pkg/server/apiextensions.go

```diff
@@ -372,13 +367,8 @@ func (c *apiBindingAwareCRDLister) getForWildcardPartialMetadata(name string) (*
 	return objs[0].(*apiextensionsv1.CustomResourceDefinition), nil
 }
 
-func (c *apiBindingAwareCRDLister) getSystemCRD(clusterName logicalcluster.Name, name string) (*apiextensionsv1.CustomResourceDefinition, error) {
```

Comment:
This particular diff looked suspicious, so I walked it back and it's been doing the same thing inside and outside of the `if` for a couple months now :)